src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/UnityCompositorInterface.cs

```diff
@@ -71,8 +71,8 @@ internal static class UnityCompositorInterface
         [DllImport(CompositorPluginDll, CharSet = CharSet.Unicode)]
         public static extern void TakeRawPicture(string path);
 
-        [DllImport(CompositorPluginDll)]
-        public static extern void StartRecording(int frameLayout);
+        [DllImport(CompositorPluginDll, CharSet = CharSet.Unicode)]
+        public static extern void StartRecording(int frameLayout, string filePath, int filePathLength);

```

Comment:
>public static extern void StartRecording(int frameLayout, string filePath, int filePathLength); [](start = 8, length = 95)

I think the recommended way to marshal an "out" string is to use a StringBuilder and not a string. String is meant to be immutable in C#, and writing directly to its buffer from native code may cause unexpected behavior. StringBuilder is meant to be a writable string buffer.

Reply:
Do you have any examples for how to do this?